akka-http/src/test/scala/fr/davit/capturl/akka/http/UriConvertersSpec.scala

```diff
@@ -127,6 +127,14 @@ class UriConvertersSpec extends FlatSpec with Matchers {
       (akkaPath: Path) shouldBe path
       (path: Uri.Path) shouldBe akkaPath
     }
+
+    // paths containing double slashes
+    {
```

Comment:
I think the fix would have to be applied even further up? Namely, in `PathParser` (which is used in `UriConverters.toAkkaUri`); this test seems to run even without any change in `UriConverters`, but calling `UriConverters.toAkkaUri(Iri("http://hello/ok//bye.html", Iri.ParsingMode.Lazy))` still results in a "Path segment must not be empty"

Reply:
You're right. I initially had the test in a `it should "convert Uri / Iri"` and there I made sure  both that the test fails without the change and that the change fixes the error. Then I decided it's supposed to go with the other path tests and didn't check again 🙈 

I've added my original test, thanks for checking!